contracts/strategies/allocation/donation-voting-open-offchain-calc/README.md

```diff
@@ -0,0 +1,75 @@
+- **Recipient information**
+    - prospective recipients must provide an address where they can receive funds
+    - prospective recipients must provide an address as `recipientId`, which serves as the unique identifier for their registration
+        - if the Allo registry is required, the `anchorId` is used is used as `recipientId`
+        - if the Allo registry is not required, the recipient has the option of using their `anchorId` or `msg.sender`
+- **Recipient eligibility**
+    - Pool manager has the option to enable two separate eligibility requirements on this contract:
+        - Requiring recipients to have an Allo registry identity in order to apply
+        - Requiring recipients to submit answers to questions, stored in metadata
+            - This metadata may include information for the front end on required / optional questions, but the contract only needs to check for whether metadata has been submitted
+            - The pool manager should have a function that enables them to edit the pool metadata, which is callable at any point
+- **Registering recipients**
+    - `registerRecipient` can be called be anyone, and is used by prospective recipients to submit an application
+        - if the recipient's application is eligible (by criteria set above), the recipient status (global and local) is set to `Pending`. If the recipient is ineligible, the transaction should revert with an error message that the application is ineligible. 
+        - pool managers must set an application start and end date on the strategy. `registerRecipient` can only be called in that window, otherwise it will revert. 
+    - Pool managers need a function to manually accept applications into the round. 
+        - If any pool manager accepts the application, the recipient status is updated to `Approved` (global and local)
+        - If any pool manager rejects the application, the recipient status is updated to `Rejected` (global and local)
+        - Pool managers are able to change an approved or rejected recipient by calling the same function. 
+    - Recipients are only able to have one registration in a pool. Re-registrations should be handled as follows:
+        - If a recipient's current status is `Pending`, then their application info is updated and their status remains `Pending` (global and local)
+        - If a recipient's current status is `Rejected`, then their application info is updated and their local status is changed to `Appealed` — global status should change to `Pending`
+        - If a recipient's current status is `Accepted`, then their application info is updated and their status is changed to `Pending` (global and local)
+
+### Allocate function logic
+In this strategy, allocators are able to express their preferences by donating tokens to the recipients they want to receive allocations. All wallets are considered eligible allocators by the contract, as actual eligibility is determined in the offchain payout calculation.
```

Comment:
last nit, change donating to allocating?

Reply:
I think donating still makes sense here. You express your allocation preference by donating

Reply:
That makes sense.